channels/ir.m3u

```diff
@@ -329,8 +437,6 @@ http://51.210.199.2/hls/stream.m3u8
 https://live.snn.ir/hls/snn/index.m3u8
 #EXTINF:-1 tvg-id="T2TV.ir" tvg-name="T2 TV" tvg-country="IR" tvg-language="Persian" tvg-logo="https://i.imgur.com/BwyQR6n.png" group-title="Music",T2 TV
 http://208.113.204.104:8123/live/tapesh-live-stream/index.m3u8
-#EXTINF:-1 tvg-id="TAMASHA.ir" tvg-name="TAMASHA" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",TAMASHA (720p)
-https://sdm.telewebion.com/live/hdtest/playlist.m3u8
```

Comment:
I almost forgot it's loading but it has returned some chunk. Needs veryfing
![image](https://user-images.githubusercontent.com/5430116/128570634-ad1db0cc-21f5-49f0-8ba5-ca6f6cc96e66.png)

Reply:
Have put 2 links above it. Had mistook with another channel, and i've fixed it. Tamasha links are under the other IRIB ones.